source/options.css

```diff
@@ -95,7 +95,7 @@ h2 ~ h2 {
 	margin-top: 1em;
 }
 
-.js-features input {
+.js-features input { /* stylelint-disable-line no-descending-specificity */
```

Comment:
Can you fix it instead?

Reply:
I found an solution for this one. Not for the other one.

Reply:
The solution for both is to move the whole rule up or down in the file. The selector doesn't need to be solved

Reply:
I think `stylelint` is awkwardly picking up the `input` selector inside the `@-moz-document` rule. Perhaps the selector fix is a better solution 👌

Reply:
I would expect stylelint to not trigger this error when the source is different, e.g. no properties matches.